### ISSUE TITLE
[Bug report] [iOS] DigitalInkRecognition: strange dot '.' recognition at the start

### ISSUE THREAD

**Issue by RogVok** [googlesamples/mlkit#614]
**Describe the bug**
Not 100% sure that that's a bug, but when the recognition process starts for the first time in an app session I see this in the console: 

> I0000 00:00:1673838707.242350       1 lstm_recognizer.cc:77] Loaded tflite model.
> I0000 00:00:1673838707.245329       1 tf_recognizer.cc:69] Loading TfRecognizer
> I0000 00:00:1673838707.246374       1 ink_preprocessor.cc:34] Created InkPreprocessor with 4 steps.
> I0000 00:00:1673838707.251791       1 tf_recognizer.cc:192] inkhash: 0ac198e7c0668d1e
> I0000 00:00:1673838707.253621       1 tf_recognizer.cc:219] LabeledInkCurveProcessor returned 39 timesteps for 0ac198e7c0668d1e.
> 2023-01-16 05:11:47.253704+0200 DigitalInkRecognitionExample[11158:3581167] Initialized TensorFlow Lite runtime.
> INFO: Initialized TensorFlow Lite runtime.
> W0000 00:00:1673838707.266690       1 tf_recognizer.cc:386] Requested external segmentation for 0ac198e7c0668d1e, but this recognizer (lang: en) does not provide segmentation.
> **I0000 00:00:1673838707.272549       1 tf_recognizer.cc:293] best result: '.'  score: 2190.51**
> I0000 00:00:1673838707.272584       1 tf_recognizer.cc:344]  1  request  8.84%    1.8ms  overall  8.84%    0.00s  LabeledInkProcessor
>  2  request 62.89%   13.1ms  overall 62.89%    0.01s  RunTfLiteModelRunner
>  3  request 28.27%    5.9ms  overall 28.27%    0.01s  Decoder
>     request          20.8ms  overall           0.02s  Total

It always recognizes a dot '.' at first. See "best result: '.'  score: 2190.51".

So is this the intended way for the DigitalInkRecognition engine to start? Or is this a bug and I should worry about it? 

Additional question: If that is intended, is it safe to force it at the app start after I initialize the recognizer like this:

> //Initialization:
> recognizer = DigitalInkRecognizer.digitalInkRecognizer(options: DigitalInkRecognizerOptions.init(model: model))
> 
> //First (empty) run which will result in a dot:
> recognize(MLKitStrokes: [Stroke.init(points: [])]) { (result: DigitalInkRecognitionResult?, error: Error?) in }

Thanks in advance.

**To Reproduce**
Example Steps to reproduce the behavior in the sample app:
1. Use the sample app from quickstarts
2. Download the needed model (en for example)
3. Write "Hello"
4. Click the recognize button
5. Notice that the engine recognized a dot before "Hello" 

**SDK Info:**
 - iOS

**Comment by miworking3** [comment]
Thanks for reporting this. The 'dot' recognition is an internal verification that the model has been loaded correctly, it is not expected to be returned from the API. Did you get the dot from the recognition result or just in the log? Thanks!

**Comment by RogVok** [comment]
I thought so. It does look like a start-up verification. 
And yes I only see that 'dot' in the log. And it is not returned from the API.

So is it OK to force it to start with an empty array like this:
`recognize(MLKitStrokes: [Stroke.init(points: [])]) { (result: DigitalInkRecognitionResult?, error: Error?) in }`
Thanks!

**Comment by RogVok** [comment]
Through trial and error, I've noticed a lag on the first use of the recognizer. That must be due to that "warm-up" process.

If I can start up the recognition engine on the app start - it won't be so noticeable to the user. So when they need the recognition later - it will be invoked faster, as the recognizer is pre-initialized.

That is what I'm trying to achieve with an empty array. Is that the correct way or not, what do you think?

**Comment by miworking3** [comment]
Yes, the recognizer is loaded "lazily" on the first recognition call, so in this case, it would be wise to "warm-up" the recognizer ahead of time. I'd recommend passing a valid ink (one that has at least one stroke, containing at least one point) to make sure that everything is actually loaded and running properly.
